x/ccv/parent/keeper/keeper.go

```diff
@@ -454,8 +454,6 @@ func (h StakingHooks) UnbondingDelegationEntryCreated(ctx sdk.Context, delegator
 	creationHeight int64, completionTime time.Time, balance sdk.Int, ID uint64) {
 	var childChainIDS []string
 
-	// TODO: once registryKeeper is implemented, we will get a list of child chains for
-	// the specific validator
 	h.k.IterateBabyChains(ctx, func(ctx sdk.Context, chainID string) (stop bool) {
```

Comment:
Here we need to iterate over _**all consumer chains registered with this provider chain**_ instead of established channels, i.e., replace `IterateBabyChains` with iterate over `ChildClient` from [here](https://github.com/cosmos/interchain-security/blob/ab20c5a401f2511398abd7bad28ea09781e1cbb6/x/ccv/parent/keeper/proposal.go#L118) (see [spec](https://github.com/cosmos/ibc/blob/marius/ccv/spec/app/ics-028-cross-chain-validation/technical_specification.md#ccv-pcf-hook-afubopcr1)). This is for safety, to ensure that unbonding operations cannot complete during channel initialization. This will partially fix issue https://github.com/cosmos/interchain-security/issues/27

Reply:
Sure, but is this relevant to this PR?

Reply:
I agree. Let's keep them separated.